src/pipecat/services/assemblyai.py

```diff
@@ -32,12 +32,12 @@ def __init__(
         self,
         *,
         api_key: str,
-        sample_rate: int = 16000,
+        sample_rate: int = 24000,
         encoding: AudioEncoding = AudioEncoding("pcm_s16le"),
         language=Language.EN,  # Only English is supported for Realtime
         **kwargs,
     ):
-        super().__init__(**kwargs)
+        super().__init__(sample_rate=sample_rate, **kwargs)
```

Comment:
There's no sample_rate in the STTService class, so we should remove this too.

Reply:
Fixed.